tox.ini

```diff
@@ -11,12 +11,15 @@ extras =
     testing
     coverage: coverage
 deps =
+    setuptools<57
     pipprevious: pip==20.3.*
     piplatest: pip
     pipmain: -e git+https://github.com/pypa/pip.git@main#egg=pip
     pip20.3: pip==20.3.*
 setenv =
     coverage: PYTEST_ADDOPTS=--strict --doctest-modules --cov --cov-report=term-missing --cov-report=xml {env:PYTEST_ADDOPTS:}
+        pipprevious,pip20.3: PIP_DISABLE_PIP_VERSION_CHECK=1
```

Comment:
those are probably not necessary - should we still use them anyways (at least the feature flag)?